crates/egui/src/menu.rs

```diff
@@ -230,6 +243,24 @@ fn stationary_menu_image_impl<'c, R>(
     InnerResponse::new(inner.map(|r| r.inner), button_response)
 }
 
+/// Build a top level menu with an text and image button.
+///
+/// Responds to primary clicks.
+fn stationary_menu_text_image_impl<'c, R>(
```

Comment:
I'm not sure we need `stationary_menu_image_impl` anymore. A normal `Button` supports both text and image, so the `ImageButton` versions of these functions shouldn't be needed anymore